configs/kubernetes

```diff
@@ -1,6 +1,6 @@
 DOCKER_REPO = gcr.io/k8s-staging-publishing-bot/k8s-publishing-bot
-NAMESPACE = k8s-publishing-bot
+NAMESPACE = publishing-bot
 SCHEDULE = * */4 * * *
 INTERVAL = 14400
 MEMORY_REQUESTS = 2Gi
-MEMORY_LIMITS = 3Gi
+MEMORY_LIMITS = 2Gi
```

Comment:
these changes seem to be legit if that's what kube needs for the deployment cc @nikhita